openbb_terminal/stocks/stocks_helper.py

```diff
@@ -1044,3 +1062,34 @@ def verify_plot_options(command: str, source: str, plot: list) -> bool:
                 )
         return True
     return False
+
+
+def heikin_ashi(data: pd.DataFrame) -> pd.DataFrame:
+    """Return OHLC data as Heikin Ashi Candles.
+
+    Parameters
+    ----------
+    data: pd.DataFrame
+        DataFrame containing OHLC data.
+
+    Returns
+    -------
+    pd.DataFrame
+        Appended DataFrame with Heikin Ashi candle calculations.
+    """
+
+    ha = candles.ha(
+        data["Open"],
+        data["High"],
+        data["Low"],
+        data["Close"],
+    )
+    ha_cols = [
```

Comment:
just make this ha.columns = [

Reply:
This serves to rename the columns, if left alone, they would be: [HA_open,HA_high, HA_low, HA_close]

Reply:
You can do it without the intermediate list.  Instead of names = [], cols = names, just do cols = []

Reply:
> You can do it without the intermediate list. Instead of names = [], cols = names, just do cols = []

I'm not sure that I follow you here.  This calls the function with the required input parameters:

```
    ha = candles.ha(
        data["Open"],
        data["High"],
        data["Low"],
        data["Close"],
    )

This renames the columns returned by the function so that when they are appended to the original DataFrame, they are presented in a similar format to the existing.

```
    ha.columns = [
        "HA Open",
        "HA High",
        "HA Low",
        "HA Close",
    ]

Is there a preferred method for renaming the columns here?